packages/core/src/Site/index.js

```diff
@@ -853,8 +851,8 @@ class Site {
     if (this.onePagePath) {
       this.mapAddressablePagesToPages(this.addressablePages || [], this.getFavIconUrl());
 
-      await this.rebuildPageBeingViewed(this.currentPageViewed);
-      await this.lazyBuildAllPagesNotViewed();
+      await this.rebuildPageBeingViewed(this.currentOpenedPages);
```

Comment:
should we do:
```suggestion
      await this._rebuildPagesBeingViewed(this.currentOpenedPages);
```
?
- plural for the expanded use case
- `_` to use the non-delayed version (possible here?)

Reply:
Agreed, I'll refactor the method (and related helpers) to use plural name, and use the non-delayed version on `rebuildSourceFiles` (as calls to it is already delayed)